test/gke/release-cluster.sh

```diff
@@ -32,14 +32,14 @@ echo "deleting terminating namespaces"
 ./delete-terminating-namespaces.sh
 
 set -e
-    
+
 echo "scaling ${cluster_uri} to 0"
 node_pools=($(gcloud container node-pools list --project "${project}" --region "${region}" --cluster "${cluster_uri}" --uri))
 if [ "${#node_pools[@]}" -ne 1 ] ; then
   echo "expected 1 node pool, found ${#node_pools[@]}"
   exit 1
 fi
 
-gcloud container clusters resize --project "${project}" --region "${region}" --node-pool "${node_pools[1]}" --num-nodes 0 --quiet "${cluster_uri}"
+gcloud container clusters resize --project "${project}" --region "${region}" --node-pool "${node_pools[0]}" --num-nodes 0 --quiet "${cluster_uri}"
```

Comment:
I wonder how it worked in the first place... also, I was under false impression that bash indexes arrays starting at 1.

Reply:
Wow, this is nuts - zsh array index starts at 1, bash starts at 0. No more comments.

Reply:
it set empty string as node-poll name, which maybe cause `gcloud` to use first one? Not sure.